x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group2/monitoring.ts

```diff
@@ -72,20 +88,30 @@ export default function monitoringAlertTests({ getService }: FtrProviderContext)
         .send(
           getTestRuleData({
             rule_type_id: 'test.patternSuccessOrFailure',
-            schedule: { interval: '3s' },
+            schedule: { interval: '1h' },
             params: {
               pattern,
             },
           })
         );
       expect(createResponse.status).to.eql(200);
-      objectRemover.add(Spaces.space1.id, createResponse.body.id, 'rule', 'alerting');
 
-      // Allow at least three executions
-      await waitForExecutionCount(5, createResponse.body.id);
+      const ruleId = createResponse.body.id;
+      objectRemover.add(Spaces.space1.id, ruleId, 'rule', 'alerting');
+
+      await waitForExecutionCount(1, ruleId);
```

Comment:
nit: could probably put these in a for loop :)

Reply:
haha that is prob a good idea! Resolved in this commit [2b150c0](https://github.com/elastic/kibana/pull/193614/commits/2b150c09ee4a27ce622c0dbb9164e70b1f2d4535)